Makefile

```diff
@@ -74,7 +74,7 @@ k3d_setup: k3d_install k3d_image helm ## install microk8s locally and deploy db-
 
 k3d_install:
 	@wget -q -O - https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash
-	@k3d cluster create myk3s -i rancher/k3s:v1.19.12-k3s1
+	@k3d cluster create myk3s -i rancher/k3s:v1.22.3+k3s1
```

Comment:
the version need to be `rancher/k3s:v1.22.3-k3s1`  ... misleading they have in the release a `+`

Reply:
Oof, thanks. I copy/pasted from the releases page. :(